src/pages/community.tsx

```diff
@@ -1,312 +1,136 @@
-  }
-`
-
-const StyledGetInvolvedCard = styled(Card)`
-  margin: 1rem;
-  padding: 1.5rem;
-  flex: 1 0 30%;
-  min-width: 280px;
-  max-width: 31%;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    max-width: 46%;
-  }
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    max-width: 100%;
-  }
-`
-
-const StyledPurpleContainer = styled.div`
-  background: ${(props) => props.theme.colors.homeBoxTurquoise};
-  display: flex;
-  align-items: center;
-  flex-direction: row-reverse;
-  padding-left: 2rem;
-  width: 100%;
-  height: 720px;
-  margin-top: -1px;
-  /* border-top: 1px solid ${(props) => props.theme.colors.text}; */
-  border-bottom: 1px solid ${(props) => props.theme.colors.text};
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column-reverse;
-    height: 100%;
-    padding-top: 2rem;
-    padding-left: 0rem;
-    padding-bottom: 2rem;
-  }
-`
-
-const StyledGrayContainer = styled(GrayContainer)`
-  box-shadow: inset 0px 0px 0px
-    ${(props) => props.theme.colors.tableItemBoxShadow};
-  padding: 0rem;
-  padding-bottom: 4rem;
-  margin-top: 0rem;
-`
-
-const StyledCardContainer = styled(CardContainer)`
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  width: 100%;
-  margin: 0rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: grid;
-    gap: 2rem;
-    grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
-    grid-template-columns: 1fr;
-  }
-`
-
-const IntroRow = styled.div`
-  display: flex;
-  align-items: center;
-  margin-bottom: 3rem;
-  margin-top: 1rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    flex-direction: column-reverse;
-    margin: 0rem;
-  }
-`
-
-const RowReverse = styled.div`
-  display: flex;
-  flex-direction: row-reverse;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column-reverse;
-    align-items: center;
-  }
-`
-
-const Row = styled.div`
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column-reverse;
-    align-items: center;
-  }
-`
-
-const CentralColumn = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  margin-bottom: 2rem;
-`
-
-const ImageContainer = styled.div`
-  background: "#f1fffd";
-  display: flex;
-  height: 100%;
-  width: 100%;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    width: 75%;
-  }
-`
-
-const IntroImage = styled(GatsbyImage)`
-  width: 100%;
-  background-size: cover;
-  background: no-repeat 50px;
-`
-
-const FeatureImage = styled(GatsbyImage)`
-  width: 100%;
-`
-
-const Subtitle = styled.div`
-  margin-bottom: 2rem;
-  font-size: 1.25rem;
-  line-height: 140%;
-  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
-    font-size: 1rem;
-  }
-`
+interface IProps {
+  children: ReactNode
+}
```

Comment:
As mentioned on a different PR (can't remember if it was one of yours xD), lets define this as
```tsx
type ChildOnlyProp = { children: ReactNode }
```
and stop using `React.FC`
```tsx
const CardContainer = (props: ChildOnlyProp) => { ... }
```

Not a blocker but a nice to have 👍🏼